test/kitchen/test/integration/win-agent-with-cws-option/rspec_datadog/win-agent-with-cws-option_spec.rb

```diff
@@ -0,0 +1,134 @@
+            expect(File).to exist("#{program_files}\\DataDog\\Datadog Agent\\bin\\agent\\ddprocmon.inf")
+        else
+            expect(is_windows_service_installed("datadog-security-agent")).to be_falsey
+            expect(is_windows_service_installed("ddprocmon")).to be_falsey
+
+            program_files = safe_program_files
+            expect(File).not_to exist("#{program_files}\\DataDog\\Datadog Agent\\bin\\agent\\ddprocmon.cat")
+            expect(File).not_to exist("#{program_files}\\DataDog\\Datadog Agent\\bin\\agent\\ddprocmon.sys")
+            expect(File).not_to exist("#{program_files}\\DataDog\\Datadog Agent\\bin\\agent\\ddprocmon.inf")
+
+        end
+    end
+    it 'does not have the driver running on install' do
+        ## verify that the driver is not started yet
+        expect(is_service_running?("ddprocmon")).to be_falsey
+    end
+
+end
+  
+shared_examples_for 'a Windows Agent with CWS running' do
+    # We retrieve the value defined in kitchen.yml because there is no simple way
+    # to set env variables on the target machine or via parameters in Kitchen/Busser
+    # See https://github.com/test-kitchen/test-kitchen/issues/662 for reference
+    let(:expect_cws_installed) {
+        parse_dna().fetch('dd-agent-rspec').fetch('cws_included')
+    }
+    it 'can start security agent' do
+        if expect_cws_installed
+            sa_conf_path = ""
+            sp_conf_path = ""
+            if os != :windows
+                sa_conf_path = "/etc/datadog-agent/security-agent.yaml"
+                sp_conf_path = "/etc/datadog-agent/system-probe.yaml"
+            else
+                sa_conf_path = "#{ENV['ProgramData']}\\Datadog\\security-agent.yaml"
+                sp_conf_path = "#{ENV['ProgramData']}\\Datadog\\system-probe.yaml"
+            end
+            expect(File).to exist(sa_conf_path)
+            expect(File).to exist(sp_conf_path)
+            
+            f = File.read(sa_conf_path)
+            confYaml = YAML.load(f)
+            if !confYaml
+                confYaml = {}
+            end
+            if !confYaml.key("runtime_security_config")
+                confYaml["runtime_security_config"] = {}
+            end
+            confYaml["runtime_security_config"]["enabled"] = true
+            File.write(sa_conf_path, confYaml.to_yaml)
+        
+            spf = File.read(sp_conf_path)
+            spconfYaml = YAML.load(spf)
+            if !spconfYaml
+                spconfYaml = {}
+            end
+            if !spconfYaml.key("runtime_security_config")
+                spconfYaml["runtime_security_config"] = {}
+            end
+            spconfYaml["runtime_security_config"]["enabled"] = true
+            File.write(sp_conf_path, spconfYaml.to_yaml)
+        
+            expect(is_service_running?("datadog-system-probe")).to be_falsey
+            expect(is_service_running?("datadog-security-agent")).to be_falsey
+            #restart "datadog-agent"
```

Comment:
Was this commented out by mistake? Implying we wanted to test the restart behavior and then proceed to stop/start? Or is it left behind commented out code that can be removed?

Reply:
it was copy paste error I removed.